src/actions/actiontype.ts

```diff
@@ -17,4 +17,6 @@ export enum ActionType {
     WaitForExternalEvent = 6,
     CallEntity = 7,
     CallHttp = 8,
+    WhenAny = 11,
+    WhenAll = 12,

```

Comment:
What happed to `9` and `10`? Should we add a comment?

Reply:
Ah, good question! `9` and `10` are `SignalEntity` and `ScheduledSignalEntity` respectively. I'll add a clarifying comment

Reply:
These comments can now be found above ^